charts/spiffe-helper-csi-driver/templates/clusterpolicy.yaml

```diff
@@ -0,0 +1,4 @@
+{{- $cp := .Files.Get "files/clusterpolicy.yaml" }}
+{{- $spiffe_helper_image := include "spire-lib.image" (dict "image" .Values.spiffeHelper.image "global" .Values.global) }}
+{{- $busybox_image := include "spire-lib.image" (dict "image" .Values.busybox.image "global" .Values.global) }}
+{{- $cp | replace "@SPIFFE_HELPER_IMAGE@" $spiffe_helper_image | replace "@BUSYBOX_IMAGE@" $busybox_image }}
```

Comment:
Why do we have to encode templating operations in a template?

Reply:
There are two template engines at play here. Helms, and Kyverno's. In order to make it easy to helm install and set the helper image to be used, it needs to run that part through helm's template engine.

Reply:
The expansion of the charts to include Kyverno is something that probably should be discussed with a majority of the maintainers.  I understand that it's part of this implementation, but its inclusion should be done with consensus, and at least some commentary on what the alternatives might be.

Reply:
Duplicated thread from above. closing this one in favor of continuing discussion above